backend/push.js

```diff
@@ -126,11 +128,14 @@ export const subscriptionInfoWrapper = (subcriptionId: string, subscriptionInfo:
         }
       })()
     },
+    'settings': {
+      value: extra.settings || {}
+    },
     'sockets': {
       value: new Set()
     },
     'subscriptions': {
-      value: new Set(channelIDs)
+      value: new Set(extra.channelIDs)
```

Comment:
`o3-mini-high` suggests:

```js
  value: new Set(extra.channelIDs || [])
```

Reply:
```
[[], null, undefined, ''].map(v => [...new Set(v)])

> (4) [Array(0), Array(0), Array(0), Array(0)]
````

So, I don't think this change is needed. It would be if maybe `extra.channelIDs` could be `false`, but I don't think it can.